src/toil/common.py

```diff
@@ -744,6 +745,8 @@ def addOptions(parser: ArgumentParser, config: Config = Config()):
     misc_options.add_argument('--forceDockerAppliance', dest='forceDockerAppliance', action='store_true', default=False,
                               help='Disables sanity checking the existence of the docker image specified by '
                                    'TOIL_APPLIANCE_SELF, which Toil uses to provision mesos for autoscaling.')
+    misc_options.add_argument('--statusWait', dest='statusWait', type=int, default=3600,
+                              help="Seconds to wait between reports of running jobs.")
```

Comment:
Where is this used?

Reply:
It gets used here: https://github.com/DataBiosphere/toil/pull/3936/files#diff-fcc766426dd77b45d1c2d6a9b79e3a34978bb849ab00412ec9085ad377a5b118R363

And then that config field is used here: https://github.com/DataBiosphere/toil/blob/c299a06fbbe67e4729a3bfcb33f66b9a5b324e69/src/toil/leader.py#L198

Reply:
Ah, it externalizes a previously existing config knob. Thanks